src/index.js

```diff
@@ -126,9 +126,13 @@ app.post('/start', (req, res) => {
   }
 
   const connection = new Connection({network: config.network, servers: [config.server], connectionTimeout: config.connectionTimeout});
+  // Previous versions of the lib would have password and pin default as '123'
+  // We currently need something to be defined, otherwise we get an error when starting the wallet
   const walletConfig = {
     seed,
-    connection
+    connection,
+    password: '123',
```

Comment:
Have we removed the default pin and password in the newer versions of the lib? I guess we should add an option to skip storing the seed and other critical data in the storage.

Reply:
Yes, we've removed the default because it's not required anymore. In the wallets, sometimes we are starting the HathorWallet object but the sensitive data is already encrypted on storage, then we don't need pin/password to create the wallet object

Reply:
Yes, we've removed the default because it's not required anymore. In the wallets, sometimes we are starting the HathorWallet object but the sensitive data is already encrypted on storage, then we don't need pin/password to create the wallet object

Reply:
Ok. For the headless wallet, these two options are not critical since the storage is just in memory.